quill-core/src/main/scala/io/getquill/dsl/MetaDsl.scala

```diff
@@ -14,8 +14,6 @@ trait MetaDslLowPriorityImplicits {
 trait MetaDsl extends MetaDslLowPriorityImplicits {
   this: CoreDsl =>
 
```

Comment:
Let's add a type alias here so the users don't necessarily need to import the type from the package:

type Embedded = io.getquill.Embedded